<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddScientist</class>
 <widget class="QWidget" name="AddScientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>157</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>589</width>
    <height>326</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="ScientistName_lineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ScientistDOB_lineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="ScientistDOD_lineEdit"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="ScientistGender_lineEdit"/>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="AddScientist_pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date of Birth (yyyy.mm.dd)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Date of Death (yyyy.mm.dd)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ScientistName_lineEdit</tabstop>
  <tabstop>ScientistDOB_lineEdit</tabstop>
  <tabstop>ScientistDOD_lineEdit</tabstop>
  <tabstop>ScientistGender_lineEdit</tabstop>
  <tabstop>AddScientist_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
